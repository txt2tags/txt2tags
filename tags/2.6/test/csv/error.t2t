
Line 1.
%!csv: error.csv
Line 2.

