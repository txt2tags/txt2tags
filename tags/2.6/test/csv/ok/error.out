txt2tags: Error: CSV: file error.csv: newline inside string
